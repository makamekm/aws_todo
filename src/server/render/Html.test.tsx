// import * as React from "react";
// import * as renderer from "react-test-renderer";
// import {
//     Html,
// } from "./Html";

// describe("Server", () => {
//     describe("Render", () => {
//         describe("Application", () => {
//             it("should call render", async () => {
//                 const html = renderer.create(<Html
//                     markup={"markup"}
//                     store={{test: "store"}}
//                     state={{test: "state"}}
//                     styles={"styles"}
//                     helmet={{
//                         base: {
//                             toComponent: jest.fn().mockReturnValue("base"),
//                         },
//                         title: {
//                             toComponent: jest.fn().mockReturnValue("title"),
//                         },
//                         meta: {
//                             toComponent: jest.fn().mockReturnValue("meta"),
//                         },
//                         link: {
//                             toComponent: jest.fn().mockReturnValue("link"),
//                         },
//                         script: {
//                             toComponent: jest.fn().mockReturnValue("base"),
//                         },
//                     }}
//                     />);

//                 expect(html.toJSON()).toMatchSnapshot();
//             });
//         });
//     });
// });
