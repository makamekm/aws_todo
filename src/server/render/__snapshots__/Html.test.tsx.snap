// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Server Render Application should call render 1`] = `
<html
  lang="en"
>
  <head>
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    base
    title
    meta
    link
    base
    styles
  </head>
  <body>
    <main
      dangerouslySetInnerHTML={
        Object {
          "__html": "markup",
        }
      }
      id="app"
    />
    <script
      charSet="UTF-8"
      dangerouslySetInnerHTML={
        Object {
          "__html": "window.__STORE_STATE__={\\"test\\":\\"store\\"};",
        }
      }
    />
    <script
      charSet="UTF-8"
      dangerouslySetInnerHTML={
        Object {
          "__html": "window.__APOLLO_STATE__={\\"test\\":\\"state\\"};",
        }
      }
    />
    <script
      src="/public/index.js"
    />
  </body>
</html>
`;
